### ISSUE TITLE
Probleme bei Eingaben in FloraDB

### ISSUE THREAD

**Issue by rebeccakurz** [barbalex/apf2#669]
Aus der Feedbackrunde: 
AgS: Bei einer Eingabe und anschliessendem Anklicken des Buttons dauert es wenige Sekunden, bis die Eingabe angenommen wird. Bei zu schneller Reaktion wird die Eingabe nicht übernommen. 

Probleme bestehen bei: Agnes Schärer, Regula Dickenmann, René Bertiller

Eventuell direkten Austausch zwischen diesen AV und Alex organisieren. Fändest du das auch sinnvoll, Alex? Vielleicht einmal eine Teams-Besprechung?

(Ich erstelle diesen Issue, damit ich es nicht vergesse)

**Comment by barbalex** [comment]
Grundsätzlich wäre es ja sowieso super gut, wenn ich Gelegenheit bekäme, AV's bei der Eingabe zuzuschauen. Da ich das noch nie machen konnte, kann es durchaus sein, dass ich dabei Überraschendes lerne.

Daher würde ich vorschlagen: Wenn jemand dieser drei offen dafür ist, würde ich gerne zu ihm/ihr vorbei gehen und folgendes machen:
- Ganz generell bei der Eigabe zuschauen
- Spezifisch dieses Problem anschauen
- Dabei auch den Blick wenden auf Sachen wie:
  - PC: Alter, Geschwindigkeit, freie Ressourcen
  - Virtuelle Betriebssysteme? (Windows auf Mac und Ähnliches)
  - Art, Qualität der Internetverbindung
  - Ob auch andere Applikationen ähnlich reagieren oder ob in diesem spezifischen Umfeld apflora eindeutig langsam auffällt

Das dauert wohl schon ein bis zwei Stunden.

Wenn ich/wir nach dem ersten Mal den Eindruck haben, dass es sich lohnt, können wir schauen, ob noch jemand dazu bereit ist. 

Ich werde versuchen, ca. zwei solche Besuche (10 Stunden) in die Offerte aufzunehmen.

Was meinst du?

**Comment by rebeccakurz** [comment]
Ja, das wäre doch eine gute Idee, wenn du mal bei jemanden oder auch verschiedenen vorbeigehen könntest und dir das mal anschauen kannst. Ich denke nicht, dass wir sonst eine Lösung finden würden, denn nur mit Beschreiben des Problems ist es wohl nicht getan. 

Nimm das also gern in die Offerte auf, vielleicht noch mit dem Hinweis, dass es an der Feedbackrunde besprochen wurde?

**Comment by barbalex** [comment]
👍

**Comment by barbalex** [comment]
Das haben wir im letzten Budget aufgenommen. Wir könnten in der nächsten Feedbackrunde informieren, dass ich bei zweien, die solche Probleme aktuell haben und demonstrieren können, vorbeikommen könnte.

**Comment by rebeccakurz** [comment]
Gerne können wir in der Feedbackrunde nächste Woche fragen, bei wem noch solche Probleme bestehen und das dann anschauen.
